contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -58,8 +58,8 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
   // tx itself, but since payment processing itself takes gas, and it needs the overhead as input, we use fixed constants
   // to account for gas used in payment processing. These values are calibrated using hardhat tests which simulates various cases and verifies that
   // the variables result in accurate estimation
-  uint256 internal constant ACCOUNTING_FIXED_GAS_OVERHEAD = 51_000; // Fixed overhead per tx
-  uint256 internal constant ACCOUNTING_PER_UPKEEP_GAS_OVERHEAD = 9_000; // Overhead per upkeep performed in batch
+  uint256 internal constant ACCOUNTING_FIXED_GAS_OVERHEAD = 51_000 + 30000; // Fixed overhead per tx
+  uint256 internal constant ACCOUNTING_PER_UPKEEP_GAS_OVERHEAD = 9_000 + 2000; // Overhead per upkeep performed in batch
```

Comment:
Hey @RyanRHall : since we modified the transmit path, the gas cost is changed. Without any changes in those constants, the typescript unit tests complains:

```
  4) AutomationRegistry2_3
       #transmit
         When the upkeep is funded
           Gas benchmarking log upkeeps [ @skip-coverage ]
             When f=10 calculates gas overhead appropriately within a margin:

Gas overhead calculated is too high, decrease account gas variables (ACCOUNTING_FIXED_GAS_OVERHEAD/ACCOUNTING_PER_SIGNER_GAS_OVERHEAD)  by at least 24874

```

I saw that you also updated those constants in a previous [PR](https://github.com/smartcontractkit/chainlink/pull/12619/commits/44c82da627de0c57cf944174ccda14e0f0dd455e), do you recommend updating those numbers based on the unit tests? Appreciate your insights~

Reply:
can we try struct packing differently and then see if we still need to increase the gas overhead? see other comment for struct packing suggestion :)